src/pages/workspace/accounting/xero/export/XeroExportConfigurationPage.tsx

```diff
@@ -22,7 +24,9 @@ function XeroExportConfigurationPage({policy}: WithPolicyConnectionsProps) {
     const menuItems: MenuItem[] = [
         {
             description: translate('workspace.xero.preferredExporter'),
-            onPress: () => {},
+            onPress: () => {
```

Comment:
@hungvu193 @lakchote I am not sure how we're handling the pendingFields here. I can see it's tagged as `policy.pendingFields.xeroConfigPreferredExporter` in the document but I don't see it being used. Neither here nor in @lakchote's PR for Bill selector.

Reply:
Yeah, I didn't find any usage of it.

Reply:
Yes, it shouldn't be `xeroConfigPreferredExporter` or anything like that. I previously wrote that before the implementation and I was wrong.

The correct way to do this is would be here to use `pendingFields?.export` for the pending action.